src/services/AuthService.js

```diff
@@ -0,0 +1,39 @@
+import { Log, UserManager } from 'oidc-client';
+
+export class AuthService {
```

Comment:
I don't think we need this class, it's just a wrapper around UserManager ?

Reply:
Yes, it initialize the userManger settings.